File: iniciante/1010/b1010.java
import java.util.Locale;
import java.util.Scanner;

public class b1010 {
  public static void main(String[] args) {
    Locale.setDefault(Locale.US);
    Scanner sc = new Scanner(System.in);

    int cod1 = sc.nextInt();
    int qtd1 = sc.nextInt();
    double preco1 = sc.nextDouble();
    int cod2 = sc.nextInt();
    int qtd2 = sc.nextInt();
    double preco2 = sc.nextDouble();

    double pagar = (qtd1 * preco1) + (qtd2 * preco2);

    System.out.println("VALOR A PAGAR: R$ " + String.format("%.2f", pagar));


    sc.close();
  }
}

File: iniciante/javascript/b1010.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var line1 = lines.shift().split(' ');
var cod1 = parseInt(line1[0]);
var qtd1 = parseInt(line1[1]);
var preco1 = parseFloat(line1[2]);

var line2 = lines.shift().split(' ');
var cod2 = parseInt(line2[0]);
var qtd2 = parseInt(line2[1]);
var preco2 = parseFloat(line2[2]);

var total = qtd1 * preco1 + qtd2 * preco2;

console.log('VALOR A PAGAR: R$ ' + total.toFixed(2));
